From: =?utf-8?q?St=C3=A9phane_Graber?= <stgraber@ubuntu.com>
Date: Thu, 15 Jan 2015 12:04:54 -0500
Subject: Add /dev/urandom to ignore_paths

systemd opens /dev/urandom during the shutdown sequence after it's done
unmounting everything. If /dev/urandom is a bind-mounted file, that will
unmount it and systemd will then crash on an assert when attempting to
read some random bytes and getting 0 back.
---
 src/core/mount-setup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/mount-setup.c b/src/core/mount-setup.c
index 6823bd8..ae68876 100644
--- a/src/core/mount-setup.c
+++ b/src/core/mount-setup.c
@@ -131,6 +131,7 @@ static const char ignore_paths[] =
         /* Container bind mounts */
         "/proc/sys\0"
         "/dev/console\0"
+        "/dev/urandom\0"
         "/proc/kmsg\0";
 
 bool mount_point_is_api(const char *path) {
